wallet-server-ui/Dockerfile

```diff
@@ -1,8 +1,8 @@
 FROM node:16-buster-slim AS builder
 
+RUN apt-get update && apt-get install -y git python3 build-essential
```

Comment:
Moving this to first makes it so Docker can reuse the cached version of the based OS install.